CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 ## Unreleased
 
 ### Added
--
+- **cf-typography:** [PATCH] Add em dash to pull quote citation.
```

Comment:
Suuuper minor, but update this entry to say "_Automatically_ add em dash …" Then good to go!